.github/workflows/require-label.yml

```diff
@@ -0,0 +1,13 @@
+name: Pull Request Labels
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mheap/github-action-required-labels@v1
+        with:
+          mode: exactly
+          count: 1
+          labels: "patch, minor, major, ignore-for-release"
```

Comment:
This config is copy-pasted from the [action's complete example](https://github.com/marketplace/actions/require-labels#complete-example) - the only thing I've changed are the label names.